dbbuild/lib/marker.py

```diff
@@ -178,6 +179,15 @@ def overlaps(self, other):
         same_chrom = self.chrom_num == other.chrom_num
         return same_chrom and self.start <= other.end and self.end >= other.start
 
+    def rsid_union(self, *others):
+        rsids = set(self.rsids)
+        for other in others:
+            rsids |= set(other.rsids)
+        for rsid in rsids:
+            for marker in chain([self], others):
+                if rsid not in marker.rsids:
+                    marker.rsids.append(rsid)
```

Comment:
Some markers are defined only using positions, no RSIDs. Some of these are exact matches for markers with some or all RSIDs defined. This function merges RSID lists when markers are merged during the build.